src/pages/api/preview.js

```diff
@@ -4,10 +4,11 @@ export default async function preview(req, res) {
     postId,
     revisionId,
     _thumbnail_id: thumbnailId,
+    token,
   } = req.query;
 
   res.setPreviewData({
-    query: { id: postId, revisionId, thumbnailId },
+    query: { id: postId, revisionId, thumbnailId, token },
```

Comment:
We are not receiving `thumbnailId` as part of the preview query

Reply:
its attached by WP